src/propcache/_helpers_py.py

```diff
@@ -27,18 +45,27 @@ def __init__(self, wrapped: Callable[..., _T]) -> None:
         self.__doc__ = wrapped.__doc__
         self.name = wrapped.__name__
 
-    def __get__(self, inst: _TSelf[_T], owner: Optional[Type[Any]] = None) -> _T:
+    @overload
+    def __get__(  # pragma: no cover
```

Comment:
@bdraco why did you add the “no cover” pragma here? It only influences Coveragepy, not MyPy. Does Coveragepy not hit this line? I'm pretty sure it should because the declaration is always executed in runtime…

Now, Coveragepy marks the line as “unmeasured” and MyPy's “partial” coverage overlays that fully, since Coveragepy does not contribute anything to the metric on this line.

Reply:
Coveragepy doesn’t hit on overloads since they are never executed at run time

Reply:
I thought this was about the function declarations. Have you seen any Coveragepy guidance on this?

Reply:
The declaration should still be executed. Then we usually have coveragerc config to ignore the `...`.

Reply:
@Dreamsorcerer yeah, my understanding is that `covdefaults` already excludes `...`.

@bdraco the concern here is that the declaration lines are excluded, and they shouldn't be. It's the body that isn't hit in runtime. But it should already be excluded automatically by the configuration we have in place.

Reply:
Removal attempt in https://github.com/aio-libs/propcache/pull/48